Gemfile-custom.sample

```diff
@@ -3,17 +3,11 @@ group :development do
   gem 'relish', '~> 0.6.0'
   gem 'guard-rspec', '~> 1.2.1'
   gem 'growl', '1.0.3'
-  gem 'spork', '0.9.0'
+  gem 'spring'
```

Comment:
This is a sample, don't encourage spring 😂

Reply:
This for sure needs to be further cleanup 😄